$NetBSD: patch-Makefile,v 1.3 2018/08/07 16:44:27 nia Exp $

Install retroarch.cfg to examples.

Don't install deprecated and unsupported cg2glsl script.

--- Makefile.orig	2018-02-19 12:43:32.000000000 +0000
+++ Makefile
@@ -201,25 +201,22 @@ install: $(TARGET)
 	rm -f $(OBJDIR)/git_version.o
 	mkdir -p $(DESTDIR)$(BIN_DIR) 2>/dev/null || /bin/true
 	mkdir -p $(DESTDIR)$(GLOBAL_CONFIG_DIR) 2>/dev/null || /bin/true
+	mkdir -p $(DESTDIR)$(PREFIX)/share/examples/retroarch 2>/dev/null || /bin/true
 	mkdir -p $(DESTDIR)$(DATA_DIR)/applications 2>/dev/null || /bin/true
 	mkdir -p $(DESTDIR)$(DOC_DIR) 2>/dev/null || /bin/true
 	mkdir -p $(DESTDIR)$(MAN_DIR)/man6 2>/dev/null || /bin/true
 	mkdir -p $(DESTDIR)$(DATA_DIR)/pixmaps 2>/dev/null || /bin/true
 	cp $(TARGET) $(DESTDIR)$(BIN_DIR)
-	cp tools/cg2glsl.py $(DESTDIR)$(BIN_DIR)/retroarch-cg2glsl
-	cp retroarch.cfg $(DESTDIR)$(GLOBAL_CONFIG_DIR)
+	cp retroarch.cfg $(DESTDIR)$(PREFIX)/share/examples/retroarch
 	cp retroarch.desktop $(DESTDIR)$(DATA_DIR)/applications
 	cp docs/retroarch.6 $(DESTDIR)$(MAN_DIR)/man6
-	cp docs/retroarch-cg2glsl.6 $(DESTDIR)$(MAN_DIR)/man6
 	cp media/retroarch.svg $(DESTDIR)$(DATA_DIR)/pixmaps
 	cp COPYING $(DESTDIR)$(DOC_DIR)
 	cp README.md $(DESTDIR)$(DOC_DIR)
 	chmod 755 $(DESTDIR)$(BIN_DIR)/$(TARGET)
-	chmod 755 $(DESTDIR)$(BIN_DIR)/retroarch-cg2glsl
-	chmod 644 $(DESTDIR)$(GLOBAL_CONFIG_DIR)/retroarch.cfg
+	chmod 644 $(DESTDIR)$(PREFIX)/share/examples/retroarch/retroarch.cfg
 	chmod 644 $(DESTDIR)$(DATA_DIR)/applications/retroarch.desktop
 	chmod 644 $(DESTDIR)$(MAN_DIR)/man6/retroarch.6
-	chmod 644 $(DESTDIR)$(MAN_DIR)/man6/retroarch-cg2glsl.6
 	chmod 644 $(DESTDIR)$(DATA_DIR)/pixmaps/retroarch.svg
 	@if test -d media/assets; then \
 		echo "Installing media assets..."; \
@@ -233,15 +230,13 @@ install: $(TARGET)
 
 uninstall:
 	rm -f $(DESTDIR)$(BIN_DIR)/retroarch
-	rm -f $(DESTDIR)$(BIN_DIR)/retroarch-cg2glsl
-	rm -f $(DESTDIR)$(GLOBAL_CONFIG_DIR)/retroarch.cfg
+	rm -f $(DESTDIR)$(PREFIX)/share/examples/retroarch/retroarch.cfg
 	rm -f $(DESTDIR)$(DATA_DIR)/applications/retroarch.desktop
 	rm -f $(DESTDIR)$(DATA_DIR)/pixmaps/retroarch.svg
 	rm -f $(DESTDIR)$(DOC_DIR)/COPYING
 	rm -f $(DESTDIR)$(DOC_DIR)/COPYING.assets
 	rm -f $(DESTDIR)$(DOC_DIR)/README.md
 	rm -f $(DESTDIR)$(MAN_DIR)/man6/retroarch.6
-	rm -f $(DESTDIR)$(MAN_DIR)/man6/retroarch-cg2glsl.6
 	rm -rf $(DESTDIR)$(ASSETS_DIR)
 
 clean:
